features/bevragen/persoon/verblijfplaats/fields-alias.feature

```diff
@@ -0,0 +1,117 @@
+    | datumVan.type        | Datum                    |
+    | datumVan.datum       | 1986-08-01               |
+    | datumVan.langFormaat | 1 augustus 1986          |
+
+    Voorbeelden:
+    | field                   | omschrijving       |
+    | datumVan                | verkorte field pad |
+    | verblijfplaats.datumVan | volledig field pad |
+
+  Scenario: consumer is geautoriseerd voor 'verblijfplaats buitenland' en vraagt het verblijfadres veld van een verblijfplaats
+    Gegeven de consumer is geautoriseerd voor 'verblijfplaats binnenland' gegevens
+    En de consumer is geautoriseerd voor 'verblijfplaats buitenland' gegevens
+    En de persoon met burgerservicenummer '000000097' heeft de volgende 'verblijfplaats' gegevens
+    | datum aanvang adres buitenland (13.20) | land adres buitenland (13.10) |
+    | 19860801                               | 5001                          |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                          |
+    | type                | RaadpleegMetBurgerservicenummer |
+    | burgerservicenummer | 000000097                       |
+    | fields              | verblijfplaats.verblijfadres    |
+    Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+    | naam | waarde                   |
+    | type | VerblijfplaatsBuitenland |
+    En heeft de 'verblijfplaats' de volgende 'verblijfadres' gegevens
+    | naam              | waarde |
+    | land.code         | 5001   |
+    | land.omschrijving | Canada |
+
+  Scenario: consumer is geautoriseerd voor 'verblijfplaats buitenland' en vraagt het land veld van verblijfadres veld van een verblijfplaats
+    Gegeven de consumer is geautoriseerd voor 'verblijfplaats binnenland' gegevens
+    En de consumer is geautoriseerd voor 'verblijfplaats buitenland' gegevens
+    En de persoon met burgerservicenummer '000000097' heeft de volgende 'verblijfplaats' gegevens
+    | datum aanvang adres buitenland (13.20) | land adres buitenland (13.10) |
+    | 19860801                               | 5001                          |
+    Als personen wordt gezocht met de volgende parameters
+    | naam                | waarde                            |
+    | type                | RaadpleegMetBurgerservicenummer   |
+    | burgerservicenummer | 000000097                         |
+    | fields              | verblijfplaats.verblijfadres.land |
+    Dan heeft de response een persoon met de volgende 'verblijfplaats' gegevens
+    | naam | waarde                   |
+    | type | VerblijfplaatsBuitenland |
+    En heeft de 'verblijfplaats' de volgende 'verblijfadres' gegevens
+    | naam              | waarde |
+    | land.code         | 5001   |
+    | land.omschrijving | Canada |
+
+Rule: de 'verblijfplaatsBinnenland' field alias moet worden gebruikt door een consumer niet is geautoriseerd voor het bevragen van 'verblijfplaats buitenland' velden
```

Comment:
'... door een consumer niet is geautoriseerd ...' --> '... door een consumer **die** niet is geautoriseerd ...'